front/migrations/20240906_2_backfill_agents_groupIds.ts

```diff
@@ -0,0 +1,155 @@
+import type { PostOrPatchAgentConfigurationRequestBody } from "@dust-tt/types";
+import { removeNulls } from "@dust-tt/types";
+import _ from "lodash";
+import type { Logger } from "pino";
+import { Sequelize } from "sequelize";
+
+import { getAgentConfiguration } from "@app/lib/api/assistant/configuration";
+import { Authenticator } from "@app/lib/auth";
+import { AgentConfiguration } from "@app/lib/models/assistant/agent";
+import { Workspace } from "@app/lib/models/workspace";
+import { DataSourceViewResource } from "@app/lib/resources/data_source_view_resource";
+import { makeScript } from "@app/scripts/helpers";
+
+makeScript({}, async ({ execute }, logger) => {
+  // All workspaces that have at least one agent
+  const workspaceIds = await getDistinctWorkspaceIds();
```

Comment:
`nit` but there is an helper to run all on workspace for next time:

https://github.com/dust-tt/dust/blob/c39e02f06cdc9dc99176c442e7d76b6d7452055a/front/scripts/helpers.ts#L80

Reply:
Thanks :pray: